chef/data_bags/crowbar/bc-template-glance.schema

```diff
@@ -76,6 +76,7 @@
             "rbd": {
               "type": "map", "required": true, "mapping": {
                 "store_ceph_conf": { "type": "str", "required": true },
+                "store_admin_keyring": { "type": "str", "required": true },
```

Comment:
I think the attribute should be required (I hate optional attributes ;-)), but I would actually make these two attributes empty by default for similar reasons @rhafer mentioned.

And again, I wouldn't rely on these attributes in the case where we want to use ceph deployed by crowbar because the user could change the attributes and break things.